File: ADS 2024.1/POO/atividades_enviar/exercicio3/q03.ts
function cumprimentarPessoa(nome: string, tratamento: string = "Sr."): void {
    console.log(`${tratamento}. ${nome}`);
}


cumprimentarPessoa("Sávia"); // Saída: Sr. Sávia
cumprimentarPessoa("João", "Dr."); // Saída: Dr. João

File: ADS 2024.1/POO/atividades_enviar/exercicio3/q03.java
public static void cumprimentarPessoa(String nome, String tratamento) {
    if (tratamento == null) {
        tratamento = "Sr.";
    }
    System.out.println(tratamento + ". " + nome);
}

public static void main(String[] args) {
    cumprimentarPessoa("Sávia", null); // Saída: Sr. Sávia
    cumprimentarPessoa("João", "Dr."); // Saída: Dr. João
}